### PR TITLE
New version: REINERSCT.cyberJackDriverPackage version 1.4.0

### PR DESCRIPTION
**Pull request by nimra98**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue? #186061

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/186060)

### CODE REVIEW

**Comment by nimra98** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 897.)

**Review by stephengillie — CHANGES_REQUESTED** [review]
> Automatic Validation ended with: Edit: image

Edit: 
Hi @nimra98,

Our pipeline uses 32-bit OSes to validate `x86` packages. 

I meant to suggest a change, but forgot to click "Start a review".

File: manifests/r/REINERSCT/cyberJackDriverPackage/1.4.0/REINERSCT.cyberJackDriverPackage.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- silentWithProgress
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.EdgeWebView2Runtime
+ElevationRequirement: elevationRequired
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://dp.reiner-sct.com/downloads/dp/dp_1_4_0.exe
+  InstallerSha256: 0F27AB139971A275EC9E8AC94D1D35C48F545854C4D0DA78D439CBB69A6558FB
```
Comment (stephengillie) [inline manifests/r/REINERSCT/cyberJackDriverPackage/1.4.0/REINERSCT.cyberJackDriverPackage.installer.yaml:20]:
```suggestion
```

**Comment by nimra98** [comment]
> > Automatic Validation ended with: Edit: image
> 
> Edit: Hi @nimra98,
> 
> Our pipeline uses 32-bit OSes to validate `x86` packages.
> 
> I meant to suggest a change, but forgot to click "Start a review".

Thanks for the help. I missed that the installer now only supports x64 Win 10 and x64/arm64 Win11, as I only copied the old entry and modified the installer location.
I also added the arm64 for Win11 and hope the pipeline runs through.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**